lib/live_sync/socket.ex

```diff
@@ -62,6 +62,28 @@ defmodule LiveSync.Socket do
     {:cont, socket}
   end
 
+  defp sync(socket, key, value, operations) do
+    operations =
+      case value do
+        nil ->
+          :delete
+
+        %{} = value ->
+          {schema, id} = LiveSync.lookup_info(value)
+
+          Enum.find_value(operations, fn {op, record} ->
+            if record.__struct__ == schema and record.id == id do
+              op
+            end
+          end)
+
+        _list ->
+          operations
+      end
+
+    socket.view.sync(key, value, socket, operations)
```

Comment:
could we have operations before socket, I was trying to keep it last to match other liveview patterns
```suggestion
    socket.view.sync(key, value, operations, socket)
```

Reply:
Changed in last commit